test/BuildingRegistry.Tests/BackOffice/Api/WhenCorrectingBuildingUnitRetirement/GivenBuildingHasInvalidStatus.cs

```diff
@@ -1,5 +1,9 @@
-namespace BuildingRegistry.Tests.BackOffice.Api.WhenCorrectingBuildingUnitRealization
+namespace BuildingRegistry.Tests.BackOffice.Api.WhenCorrectingBuildingUnitRetirement
```

Comment:
I think you moved this file and not changed/copied. So this file does not exist anymore in WhenCorrectingBuildingUnitRealization